hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/properties/ApplicationProperty.java

```diff
@@ -23,8 +23,9 @@
 	PHARMANET_ENDPOINT("pharmanet-endpoint"),
 	HIBC_E45_ENDPOINT("hibc-e45-endpoint"),
 	HIBC_R15_ENDPOINT("hibc-r15-endpoint"),
-	HIBC_R50_ENDPOINT("hibc-r50-endpoint")
-	;
+	HIBC_R50_ENDPOINT("hibc-r50-endpoint"),
+	IS_FILEDDROPS_ALLOWED("is-filedrops-allowed"),
```

Comment:
I think Wes had suggested changing this to is-filedrops-enabled instead of allowed

Reply:
You're right. That slipped my mind when reviewing the changes.

Reply:
done